setup.py

```diff
@@ -1,4 +1,4 @@
-VERSION = "3.4.2"
+VERSION = "3.4.3"
```

Comment:
As a lot of code has been moved around, improving the use of ms2pip as a library, we maybe should consider taking 3.5.0 as version number?